acceptance/suites/pre_suite/puppet4_compat/80_configure_agents.rb

```diff
@@ -0,0 +1,13 @@
+require 'puppetserver/acceptance/compat_utils'
```

Comment:
We're using the `nonmaster_agent` function. It's pretty complex.

```
def nonmaster_agents()
  agents.reject { |agent| agent == master }
end
```

Would you prefer to inline that function?